app/src/main/java/io/intrepid/thirdpartylibraryexcercise/MainActivity.java

```diff
@@ -9,9 +9,17 @@
 import android.widget.Button;
 import android.widget.ImageView;
 
-public class MainActivity extends AppCompatActivity implements MainContract.View, DownloadCatImageTask.Callback {
+import com.squareup.picasso.Picasso;
 
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+public class MainActivity extends AppCompatActivity implements MainContract.View {
+
+    @BindView(R.id.image_view)
     ImageView imageView;
+
+    @BindView(R.id.new_cat_button)
```

Comment:
instead of manually setting an onClickListener, Butterknife allows you to annotate with @OnClick on methods. This is the equivalent of setOnClickListener but much less verbose